src/main/java/ca/bc/gov/notificationservice/controller/NotificationController.java

```diff
@@ -28,6 +28,8 @@
   @PostMapping(value = "generic/{token}", produces = MediaType.APPLICATION_JSON_VALUE)
   public ResponseEntity<String> alert(@PathVariable("token") String token,
                                       @RequestBody NotificationBody notificationBody) {
+    logger.info("Received generic notification request");
```

Comment:
Do we want to include where it's coming from?

Reply:
Like interrogate the request for ip or something?

Reply:
or what alert triggered the request?